### ISSUE TITLE
Add chapter compilation support

### ISSUE THREAD

**Issue by angerhang** [mcmanigle/OxThesis#7]
At the moment, we can only compile the whole thing from scratch which is not ideal as your source files increase. It would be good to use some package like `subfiles` to allow one compile each chapter individually.

**Comment by mcmanigle** [comment]
I agree this is a great idea. I don't have the free time right now to implement it, but happy to accept a merge request if one pops up, and/or will keep it on my list for when some free time arises.
